app/views/projects/show.html.erb

```diff
@@ -63,6 +55,8 @@
                        controller: :observations,
                        action: :observations_for_project,
                        id: @project.id) %> |
+<%# FIXME This action now in UsersController; should be something like:
+  link_to :app_checklist.t users_checklist_path(params: { project_id: @project.id })%>
```

Comment:
The code suggested in this comment does not cause the test to fail.
But following the link in dev mode, throws an error.

Reply:
Hi Joe- i don't know if it makes a difference, but it looks like syntax should be  users_checklist_path(project_id: @project.id)